### ISSUE TITLE
`clusternator deploy` error

### ISSUE THREAD

**Issue by SethDavenport** [rangle/the-clusternator#307]
'clusternator deploy -d prod`

```sh
info: Requirements met, creating deployment...
/Users/sdavenpo/code/rangle/the-clusternator/node_modules/q/q.js:155
                throw e;
                ^
​
TypeError: Cannot read property 'then' of undefined
    at deploy_ (/Users/sdavenpo/code/rangle/the-clusternator/lib/api/0.1/js/js-api.js:233:76)
    at Object.deploy (/Users/sdavenpo/code/rangle/the-clusternator/lib/api/0.1/js/js-api.js:180:10)
    at /Users/sdavenpo/code/rangle/the-clusternator/lib/api/0.1/cli/deployments.js:36:17
    at _fulfilled (/Users/sdavenpo/code/rangle/the-clusternator/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/sdavenpo/code/rangle/the-clusternator/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/sdavenpo/code/rangle/the-clusternator/node_modules/q/q.js:796:13)
    at /Users/sdavenpo/code/rangle/the-clusternator/node_modules/q/q.js:604:44
    at runSingle (/Users/sdavenpo/code/rangle/the-clusternator/node_modules/q/q.js:137:13)
    at flush (/Users/sdavenpo/code/rangle/the-clusternator/node_modules/q/q.js:125:13)
    at _combinedTickCallback (node.js:370:9)
```

**Comment by bennett000** [comment]
I believe this will be fixed when @rafkhan finishes #80

**Comment by SethDavenport** [comment]
Verified.
